mailbox/plugin/quota-mailing/src/main/java/org/apache/james/eventsourcing/CommandDispatcher.java

```diff
@@ -78,10 +78,11 @@ public int getRetries() {
     }
 
     private final EventBus eventBus;
+    @SuppressWarnings("rawtypes")
     private final Map<Class, CommandHandler> handlers;
```

Comment:
Can't a `private final Map<Class, CommandHandler<?>> handlers;` do the job as well?

Reply:
Then I'm failing with a _capture-of_ in the _tryDispatch_ method.
BTW, we still need it for _Class<?>_